portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,16 +19,31 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import com.google.gson.Gson;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-    
+
+    private ArrayList<String> outputStrings = new ArrayList<String>();
```

Comment:
Make this private final List<String> greetings = new ArrayList ...

final, because you won't change the variable
Use List for the variable instead of ArrayList. List is the interface and ArrayList is the implementation. 
greetings is makes it clear what you are storing, which is more useful info to convey for code readability.

Reply:
Done